docs/workflows/alchemlyb.workflows.ABFE.rst

```diff
@@ -0,0 +1,96 @@
+The ABFE workflow
+==================
+Though **alchemlyb** is a library offering great flexibility in deriving free
+energy estimate, it also provide a easy pipeline that is similar to
+`Alchemical Analysis <https://github.com/MobleyLab/alchemical-analysis>`_ and a
+step-by-step version that allows more flexibility.
+
+Fully Automatic analysis
+------------------------
+A interface similar to
+`Alchemical Analysis <https://github.com/MobleyLab/alchemical-analysis>`_
+could be excuted with two lines of command. ::
+
+    >>> import os
+    >>> from alchemtest.gmx import load_ABFE
+    >>> from alchemlyb.workflows import ABFE
+    >>> # Obtain the path of the data
+    >>> dir = os.path.dirname(load_ABFE()['data']['complex'][0])
+    >>> print(dir)
+    'alchemtest/gmx/ABFE/complex'
+    >>> workflow = ABFE(units='kcal/mol', software='Gromacs', dir=dir,
+    >>>                 prefix='dhdl', suffix='xvg', T=298, out='./')
+    >>> workflow.run(skiptime=10, uncorr='dhdl', threshold=50,
+    >>>              methods=('mbar', 'bar', 'ti'), overlap='O_MBAR.pdf',
+    >>>              breakdown=True, forwrev=10)
```

Comment:
Add a paragraph explaining the philosophy and assumptions behind this workflow. For example, unlike other parts of alchemlyb, it finds input files inside a directory. Tell the user that certain assumptions have to hold (which ones? eg file naming?). It also sets special names for output files. What is the recommended use for the workflow, eg, input and output into the same directory?

Basically, talk a bit more about the stuff that's obvious to you but not to someone who's just been using the other parts of alchemlyb. What should you understand when using the wf?